src/utils/webrtc/analyzers/PeerConnectionAnalyzer.js

```diff
@@ -737,14 +779,32 @@ PeerConnectionAnalyzer.prototype = {
 		console.debug(tag + 'Packets per second: ' + this._packetsPerSecond[kind].toString())
 		console.debug(tag + 'Round trip time: ' + this._roundTripTime[kind].toString())
 		console.debug(tag + 'Timestamps: ' + this._timestampsForLogs[kind].toString())
+
+		this._logRtcStats(kind)
 	},
 
+	_logRtcStats(kind) {
+		const tag = this._getLogTag(kind)
+
+		for (const rtcStats of this._rtcStats[kind]) {
+			if (!rtcStats.length) {
+				console.debug(tag + 'no matching type')
```

Comment:
Dorra's now working on security hardening, so forward-thinking: it's better to not concatenate string, just `console.debug(tag, 'no matching type')` would do

Reply:
I went with fully formatted strings using `%s`, as the space between arguments is a typical implementation, but it is not explicitly required by the spec.